pkg/webhook/policy.go

```diff
@@ -51,9 +52,14 @@ var (
 func AddPolicyWebhook(mgr manager.Manager) error {
 	opa := opa.NewFromFlags()
 
+	enableManualDeploy := os.Getenv("ENABLE_MANUAL_DEPLOY")
```

Comment:
Should this be a flag option, rather than an environment variable?

We should figure out a consistent way of configuring everything to avoid fragmentation and hopefully streamline the install.

Reply:
@maxsmythe thanks for the feedback. Just updated this to use a flag instead. lmk wyt

Reply:
LGTM, thanks for doing this!

1 nit, 1 question:

nit:
If the descriptive comment is part of the commit history, we should change it to read flag instead of env variable. Dunno for sure if it would become part of the history though...

question:
It looks like we're also disabling the service. Is that intentional?

Reply:
@maxsmythe Thanks for the comment again. Updated the PR comment to reflect the most recent update. 

> It looks like we're also disabling the service

Default behavior is kubebuilder will handle the deployment of the service. If  the `enable-manual-deploy` flag is set to `true`, then users will handle the deployment of the secret, service, and the mutatingwebhookconfiguration objects.